eZ/Publish/API/Repository/Tests/SearchServiceTest.php

```diff
@@ -1701,148 +1702,171 @@ public function getSortedContentSearches()
     {
         $fixtureDir = $this->getFixtureDir();
 
-        return [
-            0 => [
-                [
-                    'filter' => new Criterion\SectionId([2]),
-                    'offset' => 0,
-                    'limit' => 10,
-                    'sortClauses' => [],
+        yield [
+            [
+                'filter' => new Criterion\SectionId([2]),
+                'offset' => 0,
+                'limit' => 10,
+                'sortClauses' => [],
+            ],
+            $fixtureDir . 'SortNone.php',
+            // Result having the same sort level should be sorted between them to be system independent
+            function (&$data) {
+                usort(
+                    $data->searchHits,
+                    function ($a, $b) {
+                        return ($a->valueObject['id'] < $b->valueObject['id']) ? -1 : 1;
+                    }
+                );
+            },
```

Comment:
You could add typehints and use spaceship operator for value comparison.

```suggestion
            function (array &$data): void {
                usort(
                    $data->searchHits,
                    function (SearchHit $a, SearchHit $b): int {
                        return $a->valueObject['id'] <=> $b->valueObject['id'];
                    }
                );
            },
```

Reply:
Fixed in 0b164f2